src/components/selection_input/selection_input.ts

```diff
@@ -46,7 +46,7 @@ interface Props {
   onSelectionReordered?: (indexes: number[]) => void;
   onSelectionRemoved?: (index: number) => void;
   onSelectionConfirmed?: () => void;
-  colors?: Color[];
+  colors?: () => Color[];
```

Comment:
I think we should rather try to have the same strategy than with `ranges`: reset it in the `onWillUpdateProps`
https://github.com/odoo/o-spreadsheet/pull/5027/commits/8ec59811c2d826c1efd3ffa9fac863d0cfd30907#diff-5cd1f7cec1d9242f20f66ef20dddedc3af5785224484dde9b55f44af90179972R134